src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -144,9 +144,9 @@ public void RemoveDataColumns(IEnumerable<DataColumn> dataColumns)
 
       public void Clear()
       {
-         _view.DeleteBinding();
          _dataColumnDTOCache.Clear();
          _allDataColumns.Clear();
+         bindToView();
```

Comment:
binding to the view instead of deleting the binding. This may avoid issue with references not being release properly